independent-publisher-connectors/EONET by NASA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,977 @@
+    "description": "The Earth Observatory Natural Event Tracker (EONET) is a prototype web service from NASA with the goal of:\n\nProviding a curated source of continuously updated natural event metadata.\nProviding a service that links those natural events to thematically-related web service-enabled image sources (e.g., via WMS, WMTS, etc.).",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "eonet.sci.gsfc.nasa.gov",
+  "basePath": "/api/v3/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/events": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "title": {
+                  "type": "string",
+                  "description": "The title of the API endpoint.",
+                  "title": "Title"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "The description of the response.",
+                  "title": "Response Description"
+                },
+                "link": {
+                  "type": "string",
+                  "description": "The full link to the API endpoint.",
+                  "title": "Endpoint Link"
+                },
+                "events": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Unique ID for this event.",
+                        "title": "ID"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "The title of the event.",
+                        "title": "Event Title"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Optional longer description of the event.",
+                        "title": "Description"
+                      },
+                      "link": {
+                        "type": "string",
+                        "description": "The full link to this specific event.",
+                        "title": "Event Link"
+                      },
+                      "closed": {
+                        "type": "string",
+                        "description": "An event is deemed \u201cclosed\u201d when it has ended. The closed field will include a date/time when the event has ended. Depending upon the nature of the event, the closed value may or may not accurately represent the absolute ending of the event. If the event is open, this will show \u201cnull\u201d.",

```

Comment:
@troystaylor Unicodes - The connector's swagger has unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in English language.

Reply:
Done